test/city-council-district/city-council-district.e2e-spec.ts

```diff
@@ -54,4 +57,45 @@ describe("City Council District e2e", () => {
       await app.close();
     });
   });
+
+  describe("findCapitalProjectsByCityCouncilDistrictId", () => {
```

Comment:
please refactor to account for limits and offsets, similar to [tax lot tests](https://github.com/NYCPlanning/ae-zoning-api/blob/main/test/tax-lot/tax-lot.e2e-spec.ts#L36)